packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -93,7 +101,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     return (
       <div ref='root'>
         { label && (
-          <label id={ id + '-label' } className='ms-Label' ref={ (dropdownLabel) => this._dropdownLabel = dropdownLabel } >{ label }</label>
+          <Label id={ id + '-label' } ref={ this._resolveRef('_dropdownLabel') } required={ required }>{ label }</Label>
```

Comment:
Yeap, I saw `htmlFor` in your PR, but I think that using `for` attribute is not correct in this case. **Dropdow** is a `div` container and `div` is not _labelable_ element. Use of `for` attr does not give any advantages. See more [here](http://stackoverflow.com/a/11705966).